OpenRA.Mods.Common/Widgets/Logic/Editor/ActorSelectorLogic.cs

```diff
@@ -202,6 +210,16 @@ public ActorSelectorLogic(Widget widget, World world, WorldRenderer worldRendere
 			InitializeActorPreviews();
 		}
 
+		void SelectOwner(PlayerReference option)
+		{
+			selectedOwner = option;
+
+			ownersDropDown.Text = selectedOwner.Name;
```

Comment:
Style nit: please use `option` here instead of `selectedOwner` for consistency.

Reply:
This was copy pasted over from up top btw, this code already existed. (Still worth changing it)